code/index.html

```diff
@@ -3,28 +3,296 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Karla&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/59fae56e7e.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Guldfabriken Studios</title>
   </head>
+
+  <!-- ----------Video hero and img logo---------- -->
```

Comment:
these are great for organizing everything! makes it easy to navigate

Reply:
Thanks Elsa!